### DISCUSSION TITLE
Output single Lua file

### DISCUSSION THREAD

**Discussion by coelhucas** [teal-language/tl#738] [Q&A] [answered]
Is it possible to do so? Thanks!

**Comment by hishamhm** [comment, marked as answer]
Not sure what you mean by single, but each .tl file compiles to one .lua file (for example, using `tl gen` )

**Reply by coelhucas** [reply]
Literally a single file. Wanted to generate a bundle with everything merged to a single file. So it's not possible so far, thanks!

**Reply by hishamhm** [reply]
After you compile your .tl files to .lua you can use one of the various existing Lua projects for bundling, such as luapak!

**Reply by catwell** [reply]
If you need something simpler that just deals with combining several Lua files into one, check out [LuaCC](https://github.com/mihacooper/luacc).
